### PR TITLE
Add default path

### PR DESCRIPTION
**Pull request by jfmengels**
- Fix error when specifying no path.
  When pressing enter for the `files` question, the value of `answer.files` is `['']`, which passes the `files.length` condition.
- Check the `package.json` for AVA config to get the `files` path and use it as the default path. Use AVA's default otherwise.

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -72,6 +73,9 @@ codemods.sort(utils.sortByVersion);
 
 var versions = utils.getVersions(codemods);
 
+var avaConf = pkgConf.sync('ava');
```
Comment (jamestalmage) [inline cli.js:80]: What does pkgConf do if there is no `package.json` ?

Comment (jamestalmage) [inline cli.js:80]: Also, need to handle possibility `ava` section is not defined.

Comment (jfmengels) [inline cli.js:80]: It returns an empty object from what I can test.

Comment (jfmengels) [inline cli.js:80]: > Also, need to handle possibility ava section is not defined.

That's handled well. If there's no package, if the `ava` section is empty/non-existent, then it all evaluates to `{}`, and files will then take the `defaultFiles` value.

Comment (jfmengels) [inline cli.js:80]: Yup, confirming, seems to work fine without a `package.json` file, and any other configuration I mentioned above.

File: cli.js [hunk 2]
```diff
@@ -86,9 +90,12 @@ var questions = [{
 	type: 'input',
 	name: 'files',
 	message: 'On which files should the codemods be applied?',
+	default: (avaConf.files && avaConf.files.join(' ')) || defaultFiles,
```
Comment (jamestalmage) [inline cli.js]:
```
avaConf.files && arrify(avaConf.files).join(' ')
```

Users are not required to specify an array. It can be just a single string.

Comment (jamestalmage) [inline cli.js]: I fixed it.

**Pull request merged**